browser/brave_rewards/extension_rewards_service_observer.cc

```diff
@@ -221,7 +222,7 @@ void ExtensionRewardsServiceObserver::OnPendingContributionSaved(
 
 void ExtensionRewardsServiceObserver::OnPublisherListNormalized(
     RewardsService* rewards_service,
-    const brave_rewards::ContentSiteList& list) {
+    ledger::PublisherInfoList list) {
```

Comment:
Should still be a const reference, IMO (i.e., `const ledger::PublisherInfoList&`)

Reply:
fixed